File: typeScriptCodeWars/test/array/eight/arrayPlusArray.test.ts
const {arrayPlusArray} = require('../../../src/array/eight/arrayPlusArray')

test("Test array plus array", () => {
    expect(arrayPlusArray([3, 3, 4], [3, 3, 4])).toEqual(20);
    expect(arrayPlusArray([-1, -2, -3], [-4, -5, -6])).toEqual(-21);
    expect(arrayPlusArray([-1, -2, -3, -4], [10])).toEqual(0);
})

File: javaCodeWars/javaCodewars/src/test/java/com/zh/sergei/codewars/arrays/eight/ArrayPlusArrayTest.java
package com.zh.sergei.codewars.arrays.eight;

import org.junit.jupiter.api.Test;
import static org.assertj.core.api.Assertions.assertThat;
public class ArrayPlusArrayTest {

    @Test
    public void arrayPlusArray_summElementsOfTwoProvidedArrays(){
        assertThat(ArrayPlusArray.arrayPlusArray(new int[]{1,2,3}, new int[]{4,5,6})).isEqualTo(21);
        assertThat(ArrayPlusArray.arrayPlusArray(new int[]{-1,-2,-3}, new int[]{-4,-5,-6})).isEqualTo(-21);
    }
}
